src/main/java/dev/projectg/geyserhub/module/bossBar/BossBarHandler.java

```diff
@@ -0,0 +1,27 @@
+package dev.projectg.geyserhub.module.bossBar;
+
+import dev.projectg.geyserhub.GeyserHubMain;
+import dev.projectg.geyserhub.config.ConfigId;
+import org.bukkit.configuration.file.FileConfiguration;
+
+import java.util.HashMap;
+import java.util.Objects;
+
+public class BossBarHandler {
+
+    public static HashMap<Integer, String> color = new HashMap<>();
+    public static HashMap<Integer, String> text = new HashMap<>();
+    public static HashMap<Integer, String> style = new HashMap<>();
+
+    public void bossBarSetup(){
+        FileConfiguration config = GeyserHubMain.getInstance().getConfigManager().getFileConfiguration(ConfigId.MAIN);
+
+        int i = 1;
+        while(config.getString("BossBar.Bar" + i + ".Text") != null) {
```

Comment:
Can we this this as the expression for the while loop:

`config.contains("BossBar.Bar" + i + ".Text") && config.contains("BossBar.Bar" + i + ".Color") && `

Reply:
Also, should we be enforcing the keys to all be of the format ("Bar"+int) ? Does it really matter if they aren't?

Reply:
Think we can yea, I'm still not sure if we should keep the loop who actually want 2 bossbars :p. But incase you want to keep it, for enforcement don't think is would matter that much, You can change it if you want to